### DISCUSSION TITLE
Chapter nibbles on web videos affected by seek range options

### DISCUSSION THREAD

**Discussion by Keith94** [Samillion/ModernZ#243] [Q&A] [answered]
seekrangealpha and seekbar_cache_color are affecting both the cache bar and the seekbar nibbles. Is it supposed to do that?

![5pk9FMk5e3](https://github.com/user-attachments/assets/3f32e7bb-f357-4fdc-b819-bb9795d800f7)

**Comment by Samillion** [comment, marked as answer]
Terribly sorry about the late response, not sure why I wasn't notified of this discussion.

Nibbles inherits the seekbar style. I honestly haven't looked thoroughly into it, but I remember seeing it in the past.

**Comment by Keith94** [comment]
No worries, it happened to me as well before over in #212 😄 GitHub bug?

Anyways, if it's intended, should be fine.

Initially it just seemed odd that cache color was overriding the foreground color for nibbles, and only for web videos, too (obviously).
